src/plugin/transform.ts

```diff
@@ -92,18 +100,19 @@ export function transform({
         mdDir.renderer.rules[TOKEN_TYPE] = (tokens, idx, _opts, _env, self) => {
             const token = tokens[idx];
 
-            const base = `<base target="${baseTarget}">`;
-            let content = base + token.attrGet(TokenAttr.srcdoc) ?? '';
-
+            let additional = baseTarget ? `<base target="${baseTarget}">` : '';
             if (styles) {
                 const stylesContent =
                     typeof styles === 'string'
                         ? `<link rel="stylesheet" href="${styles}" />`
                         : `<style>${getStyles(styles)}</style>`;
-                content = stylesContent + content;
+                additional += stylesContent;
             }
 
-            const resultContent = sanitize ? sanitize(content) : content;
+            const headContent = head ? `<head>${head}</head>` : additional;
```

Comment:
In the previous version, additional content was added to the main body. It could have been placed in the header, but for compatibility reasons, I decided to keep it in the main content.